src/main/kotlin/no/nav/tilleggsstonader/sak/statistikk/behandling/dto/DvhEnums.kt

```diff
@@ -0,0 +1,17 @@
+package no.nav.tilleggsstonader.sak.statistikk.behandling.dto
+
+enum class Hendelse {
+    MOTTATT,
+    PÅBEGYNT,
+    VENTER,
+    VEDTATT,
```

Comment:
Er disse hendelsene noe dvh angir? Uten å ha sett denne lista ville jeg tenkt at `VEDTATT` er det samme som `BESLUTTET`, så mulig det er litt potensiale for å misforstå begreper 😅  Men helt greit hvis det er dvh som ønsker disse enums

Reply:
Hmm ja, eller er det sånn at vedtatt er når saksbehandler 1 sender til totrinnskontroll?

Reply:
Enumen er det samme som EF sender til DVH. VEDTATT betyr i praksis at saksbehandler sender behandlingen videre til totrinnskontroll.